src/client/main.c

```diff
@@ -0,0 +1,71 @@
+#include <errno.h>
+#include <getopt.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "client.h"
+#include "libhirte/common/common.h"
+#include "libhirte/common/opt.h"
+
+const struct option options[] = { { ARG_HELP, no_argument, 0, ARG_HELP_SHORT }, { NULL, 0, 0, '\0' } };
+
+#define OPTIONS_STR ARG_HELP_SHORT_S
+
+static char *op;
+static char **opargv;
+static int opargc;
+static bool no_action = false;
+
+static void usage(char *argv[]) {
+        fprintf(stderr, "Usage: %s COMMAND ... \n", argv[0]);
+}
+
+static int get_opts(int argc, char *argv[]) {
+        int opt = 0;
+
+        while ((opt = getopt_long(argc, argv, OPTIONS_STR, options, NULL)) != -1) {
+                if (opt == ARG_HELP_SHORT) {
```

Comment:
Could we change this to a `switch-case`? I know, currently it doesn't make that much sense, but we will probably add options in the future.

Reply:
Actually the linter shouts on me when I make that a switch-case :)